starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -441,6 +441,21 @@ List<Property> properties
 @if (features.contains("jmx")) {
 @dependency.template("io.micronaut.jmx", "micronaut-jmx", "compile", null)
 }
+@if (features.contains("dekorate-kubernetes")) {
+@dependency.template("io.dekorate", "kubernetes-annotations", "compile", "1.0.0")
+}
+@if (features.contains("dekorate-openshift")) {
+@dependency.template("io.dekorate", "openshift-annotations", "compile", "1.0.0")
+}
+@if (features.contains("dekorate-knative")) {
+@dependency.template("io.dekorate", "knative-annotations", "compile", "1.0.0")
```

Comment:
I'll remove the versions once the https://github.com/micronaut-projects/micronaut-core/pull/4063 is merged